ui/src/job/form/context.js

```diff
@@ -209,6 +210,26 @@ export const JobFormContextProvider = ({ job: initJob, ...props }) => {
     },
     [setJob]
   );
+  
+  // This port partial of the FormContextProvider that most of the forms are using,
+  // so that the similiar/future section using onChangeHandler can be reused and 
+  // does not requires adding new callbacks per field
+  var handleChanges = React.useCallback(function (paths, value) {
```

Comment:
i totally get that why we -should- do this, just that I was concern over refactoring the other 10 callbacks created in this file as a result of not using the `FormContextProvider` and this was my half attempt at using the `FormContextProvider` implementation just for this new field instead of adding another callback 😅 

i'll take a stab at this tomorrow or in a separate PR.

Reply:
That's fair. Perhaps we can wrap the `JobForm` with the `FormContextProvider` (instead of defining the generic callback handlers here) but only take advantage of it for the new fields first and add a TODO in this file for refactoring the existing ones.

Anyway, if this proves to be troublesome, we can just add the TODO, keep things as is and revisit this later too.